turbo.json

```diff
@@ -27,7 +27,10 @@
     },
     "//#build": {
       "cache": false,
-      "dependsOn": []
+      "dependsOn": [],
+      "inputs": [
+        "**"
```

Comment:
We seems to have `cache: false` on `//#build`, why do we still need `inputs: "**"`?